crates/bevy_math/src/cubic_splines.rs

```diff
@@ -640,6 +645,7 @@ pub trait CubicGenerator<P: VectorSpace> {
 ///
 /// Segments can be chained together to form a longer compound curve.
 #[derive(Copy, Clone, Debug, Default, PartialEq)]
+#[cfg_attr(feature = "reflect", derive(bevy_reflect::Reflect), reflect(Debug, Default))]
```

Comment:
You’ll need to conditionally import the `ReflectDefault` type data.

The same is true for other type data like `ReflectSerialize` and `ReflectDeserialize`.